libraries/AP_Airspeed/AP_Airspeed_MS4525.cpp

```diff
@@ -170,8 +193,10 @@ void AP_Airspeed_MS4525::_collect()
     float temp  = _get_temperature(dT_raw);
     float temp2 = _get_temperature(dT_raw2);
     
-    _voltage_correction(press, temp);
-    _voltage_correction(press2, temp2);
+    if (!disable_voltage_correction()) {
```

Comment:
Do these not need correcting, or is it just the board voltage is an incorrect voltage source so does more harm than good?

Reply:
if the sensor has an LDO then correcting based off board voltage is incorrect